### DISCUSSION TITLE
Notification Sounds Don’t Work on iOS

### DISCUSSION THREAD

**Discussion by mikaelacaron** [community/community#69612] [Mobile]
I don't hear a notification sound when a new GitHub notification comes in. I only see it as a banner and in the notification list.

I tried to turn off notifications and turn them on again, but still nothing. My phone is not in do not disturb.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.133.1 (112099449)
Device: iPhone15,2 (iOS 17.0.3)
</details>

**Comment by Mohammed71o** [comment]
https://github.com/GoogleChrome/developer- .chrome.com/pull/7484

**Comment by elisealix22** [comment]
Hi @mikaelacaron , thank you for the report! I added an internal issue to track this with our iOS team. Please let us know if you have any other suggestions to make the app better in the meantime!
